ros_buildfarm/templates/snippet/builder_system-groovy_check-for-failing-jobs.xml.em

```diff
@@ -0,0 +1,82 @@
+@(SNIPPET(
+    'builder_system-groovy',
+    command=
+"""// CHECK FOR FAILING JOBS
+import hudson.model.Job
+
+// Control the number of failures in a row to check
+min_number_of_failures = 5
+// Control the minimun amount of time failing
+min_days_failing = 7
+// rosdistro to search
+rosdistro = "@ros_distro"
+
+boolean is_failing_from_n_days(int min_days_failing,
+                               Job job)
+{
+  now = new Date()
+  start_of_failing_period = now - min_days_failing
+
+  last_success_build = job.getLastSuccessfulBuild()
+  
+  // if there is a last success build check when it was  
+  if (last_success_build)
+  {
```

Comment:
Please keep the open `{` trailing the previous line within a function (for `if`, `while`, etc.).

Same below.